doc/tech/usecases.md

```diff
@@ -30,8 +30,46 @@
 <a name="5.1.5"></a>
 ### 5.1.5 Use cases
```

Comment:
Can't we just refer to [OSF ECG-s Use Case list](https://wiki.openstack.org/wiki/Edge_Computing_Group/Use_Cases)? If we would like to add something new or change anything on the original list, let's just bring it to ECG. @bfcohen, @ildikov and myself can help in that.

Reply:
@CsatariGergely , as per first meeting , we agreed that @Karimrabie will scan OSF ECG use case with OPNFV cases and select the suitable one after organizing it 

and it presented last meeting and we agreed to move with the 4-5 use cases that attached in the slides 

@CsatariGergely , make sense ?

Reply:
>
> and it presented last meeting and we agreed to move with the 4-5 use cases that attached in the slides

@ASawwaf Is that material going to be put in this PR? Right now it's just a shell.

Reply:
@markshostak , we supposed to have the above material to be added in PR, we tried yesterday but we failed :( to add it  , so I will try today, it will great if you can support @Karimrabie

Reply:
> @markshostak , we supposed to have the above material to be added in PR, we tried yesterday but we failed :( to add it , so I will try today, it will great if you can support @Karimrabie

@ASawwaf @markshostak Let me try to do that today. 
I will populate some content under the use cases section and we can take it from there.

@CsatariGergely 
The methodology that we discussed in our previous call is to link the technical use cases to a practical 5G Business Service that operators are keen to realise.
So, instead of CDN as a use case, we go for an **Edge eMBB Network Slice with CDN**.

This is in addition to the quick wins such as **vRAN**. Which is implicitly an ingredient of a custom commercial network slice service as well.

We will make use of whatever been achieved in OSF ECG and that's actually what I did by scanning the use cases and identifying the use cases that we can build on top of it.